ui/app/AppLayouts/WalletV2/components/AddAccountPopup.qml

```diff
@@ -0,0 +1,211 @@
+                    font.weight: Font.Medium
+                    font.pixelSize: 13
+                    color: Style.current.textColor
+                }
+                Rectangle {
+                    width: parent.width
+                    height: 56
+                    anchors.top: inputLabel.bottom
+                    anchors.topMargin: 7
+                    radius: 10
+                    color: "pink"
+                    opacity: 0.6
+                }
+            }
+        }
+
+        StatusInput {
+            id: passwordInput
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 10
+            implicitHeight: 82
+            input.implicitHeight: 56
+            anchors.top: accountNameInputRow.bottom
+            anchors.topMargin: popup.marginBetweenInputs
+            input.placeholderText: qsTr("Give the account a secure password")
+            label: qsTr("Password")
+            input.echoMode: TextInput.Password
+            validators: [StatusMinLengthValidator { minLength: 6 }]
+            onTextChanged: {
+                errorMessage = (passwordInput.text === "") ?
+                            qsTrId("you-need-to-enter-a-password") :
+                            qsTrId("password-needs-to-be-6-characters-or-more");
+            }
+        }
+    }
+
+    rightButtons: [
+        StatusButton {
+            text: popup.loading ? qsTrId("loading") : qsTrId("add-account")
+            enabled: !popup.loading && (passwordInput.text !== "") && (accountNameInput.text !== "")
+                     && (keyOrSeedPhraseInput.correctWordCount || (keyOrSeedPhraseInput.text !== ""))
+
+            MessageDialog {
+                id: accountError
+                title: qsTr("Adding the account failed")
+                icon: StandardIcon.Critical
+                standardButtons: StandardButton.Ok
+            }
+
+            onClicked : {
+                popup.loading = true;
+                if (!popup.validate()) {
+                    errorSound.play();
+                    popup.loading = false;
+                } else {
+                    //TODO account color to be verified with design
+                    const result = popup.seedPhraseInserted ?
+                                     walletModel.accountsView.addAccountsFromSeed(keyOrSeedPhraseInput.text, passwordInput.text, accountNameInput.text, "") :
+                                     walletModel.accountsView.addAccountsFromPrivateKey(keyOrSeedPhraseInput.text, passwordInput.text, accountNameInput.text, "");
+                    popup.loading = false;
+                    if (result) {
+                        let resultJson = JSON.parse(result);
+                        if (Utils.isInvalidPasswordMessage(resultJson.error)) {
+                            passwordInput.valid = false;
+                            passwordInput.errorMessage = qsTrId("wrong-password");
```

Comment:
```suggestion
                            passwordInput.errorMessage = qsTr("wrong-password");
```

Reply:
I don't quite get this, in case I replace `qsTrId` with `qsTr`, I would also have to remove dashes and add upper case first letter so to look like `qsTr("Wrong password")`. `qsTr("wrong-password")` would display "wrong-password". Nevertheless I don't see any misbehaviour as it is currently, do you see any issues on your end?